index.html

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <script
+      src="https://kit.fontawesome.com/e495230f5d.js"
+      crossorigin="anonymous"
+    ></script>
+    <link rel="stylesheet" href="style.css" />
+    <title>Mint Signup form</title>
+  </head>
+  <body>
+    <nav>
+      <p class="already">Already have an Intuit account?</p>
+      <a
+        class="forgot"
+        href="https://accounts.intuit.com/account-recovery.html"
+        >I forgot my User ID or Password</a
+      >
+      <button class="signin">
+        <i class="fa fa-lock" aria-hidden="true"></i>Sign In
+      </button>
+    </nav>
+
+    <main>
+      <div class="branding">
+        <ul>
+          <li class="intuit"></li>
+          <li class="mint"></li>
+          <li class="quickbooks"></li>
+          <li class="turbotax"></li>
+        </ul>
+      </div>
+      <section class="signup">
+        <h3>Create an Intuit account</h3>
+        <p>One account for everything Intuit, including Mint.</p>
+        <a
+          href="https://accounts-help.lc.intuit.com/questions/1582580-creating-an-account"
+          class="learn-more"
+          >Learn more</a
+        >
+        <form action="#" class="signup-form">
+          <label for="email">Email Address</label>
+          <input type="email" name="email" id="email" />
+          <label for="phone">Phone (recommended)</label>
+          <input type="tel" name="phone" id="phone" />
+          <p>Standard call, messaging or data rates may apply.</p>
+          <label for="password">Password</label>
+          <input type="password" name="password" id="password" />
+          <button class="signin create-btn">
+            <i class="fa fa-lock" aria-hidden="true"></i>Create Account
+          </button>
```

Comment:
The button should have a type='submit':
- [ ] It has a `<submit>` button that matches the style of the original page (Blue background, white text) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/html_forms/form_and_input_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59524927/))